### DISCUSSION TITLE
Circuit Sword Software version 1.4.1

### DISCUSSION THREAD

**Discussion by weese** [weese/Circuit-Sword#4] [Announcements]
* based on [RetroPie 4.8](https://retropie.org.uk/2022/03/happy-pi-day-retropie-4-8-is-released/), Raspbian kernel 5.15.34
* **2022-04-26 UPDATE:** [FIX] Bluetooth wasn't working with Retropie's default cmdline.txt, see [discussion](https://github.com/weese/Circuit-Sword/discussions/2)
*  **2022-04-26 UPDATE:** DKMS modules come pre-installed to survive kernel updates, tested with `apt-get upgrade raspberrypi-kernel raspberrypi-kernel-headers`

<hr /><em>This discussion was created from the release <a href='https://github.com/weese/Circuit-Sword/releases/tag/v1.4.1'>Circuit Sword Software version 1.4.1</a>.</em>

**Comment by weese** [comment]
[@Antho91](https://github.com/Antho91) I finally integrated the pre-installed DKMS modules you wanted 😊

**Reply by Antho91** [reply]
Nice, that's so awesome. Now, users don't have to enter commands for keeping the sword's Wi-Fi and other drivers working after an update ;).
